BusinessTampereTrafficMonitoring/traffic_lights/signal_group.py

```diff
@@ -44,7 +69,7 @@ def update_state(self, timestamp: str, status_code: str):
             # were multiple state changes during the polling interval
             if self.t_red_start is not None:
                 self.t_green_start = timestamp
-            return None
+                return None
```

Comment:
This should not have been changed. The return was intended to be outside the if statement. The code still works the same, but only because functions in Python return `None` by default if there is no explicit return statement and this happens to run to the end of the function body. If the code was expanded later this would result in hard-to-find bugs.